components/mlmd/Get_artifact_types/component.yaml

```diff
@@ -0,0 +1,77 @@
+name: Get artifact types from mlmd
+description: Gets artifact types from MLMD
+outputs:
+- {name: Output, type: JsonArray}
+implementation:
+  container:
+    image: python:3.7
```

Comment:
I wonder if there's anything that prevent us from using python-alpine as default. That should make the image a lot smaller.

Reply:
I'm not sure it will work as the default. It uses a `musl` library instead of `glibc`, which can lead to some compatibility issues. See "Image variants" section in https://hub.docker.com/_/python

I was surprised to see the size difference. I suspect that most size diff comes from something other than the base OS image. Maybe some python modules are cut like in slim.

Anyways, I think this topic might be worth exploring, but for now the default official image ( `docker pull python`) might be a preferred choice.

Reply:
I see, https://pythonspeed.com/articles/alpine-docker-python/ is a good reference, prebuilt python binary wheels may not work well on alpine.